src/mixins/bcol-mixin.ts

```diff
@@ -20,7 +20,9 @@ export default class BcolMixin extends Vue {
 
   getErrorCode (error: any): string {
     if (error?.response?.data?.errors) {
-      const msgCode = error.response.data.errors.find(x => x.payment_error_type && x.payment_error_type.startsWith('BCOL'))
+      const msgCode = error.response.data.errors.find(x =>
+        x?.payment_error_type?.startsWith('BCOL')) // eslint-disable-line camelcase
```

Comment:
FYI, another way to add the lint exception for a long line is to specify is before the line with the command "eslint-disable-next-line ...".